ios/TabViewImpl.swift

```diff
@@ -76,6 +77,9 @@ struct TabViewImpl: View {
         UITabBar.appearance().scrollEdgeAppearance = configureAppearance(for: newValue ?? "")
       }
     }
+    .onAppear {
```

Comment:
You should use `.onChange(of: props.barTintColor)` to allow dynamic changing.

Reply:
Thank you for the suggestion!

I initially tried using .onChange(of: props.barTintColor), but it doesn't seem to update the barTintColor as expected in my case. The color only reflects properly when using .onAppear.

To make sure the `barTintColor` reflects both on view load and during dynamic changes, I combined both .onAppear and .onChange(of:). This approach ensures that the UITabBar appearance is set correctly when the view first appears and updates dynamically when barTintColor changes.

Here is what I am going to change:

```
.onAppear {
    updateTabBarAppearance(with props.barTintColor)
}
.onChange(of: barTintColor) { newColor in
    updateTabBarAppearance(with newColor)
}
```

Is this okay? @okwasniewski

Reply:
If thats the case you can do a custom extension on the view, lets keep simillar logic contained:

```swift
extension View {
  @ViewBuilder
  func tabBarTintColor(enabled: Bool) -> some View {
    self
		.onAppear {
            // Do something
         }
       .onChange(of: barTintColor) { newColor in
    // Do something
      }
  }
  ```